documentation/src/main/asciidoc/topics/properties/hotrod-client-remote.properties

```diff
@@ -0,0 +1,2 @@
+infinispan.client.hotrod.auth_username=admin
```

Comment:
I would add `infinispan.client.hotrod.server_list = 127.0.0.1:11222` here so we can have a basic working config.

Reply:
Sure. Thanks @pruivo